apps/doc-gen/src/app/docgen-client.ts

```diff
@@ -45,19 +48,22 @@ export class DocGenClient extends PluginClient {
         input: input,
         output: output
       }
-      this.fileName = fileName
-      this.eventEmitter.emit('compilationFinished', this.build, fileName)
+      const test = normalizeContractPath(fileName, true)
+      this.fileName = typeof test === 'string' ? test : test[test.length - 1]
```

Comment:
`this.fileName` in that case doesn't contain `.sol`. This shouldn't eb the case.

Reply:
I have since fixed this also.